pootle/apps/pootle_store/models.py

```diff
@@ -248,51 +270,51 @@ def get_translatable(self, user, project_code=None, language_code=None,
            from the TP.
         :param filename: A string for matching the filename of Stores.
         """
-        from pootle_project.models import Project
+        from pootle.core.site import pootle_site
 
-        if not user.is_superuser:
-            user_projects = Project.accessible_by_user(user)
-            if project_code and project_code not in user_projects:
-                return self.none()
-
-        units_qs = self.get_for_user(user)
+        units_qs = self.get_for_user(user, project_code=project_code)
 
         if language_code:
             units_qs = units_qs.filter(
-                store__translation_project__language__code=language_code)
-        else:
-            units_qs = units_qs.exclude(
-                store__pootle_path__startswith="/templates/")
-
-        if project_code:
+                language_id=pootle_site.languages[language_code]["pk"])
+        elif project_code:
             units_qs = units_qs.filter(
-                store__translation_project__project__code=project_code)
-        elif not user.is_superuser:
+                language_id__in=pootle_site.projects[project_code]["languages"])
```

Comment:
Is it not overcomplicating it? I mean, we are filtering units here, so why not:

```python
            units_qs = units_qs.filter(project_id=project_code)
``` 

Perhaps `project_id__in` in my first comment was the root of the confusion. My apologies.